src/Column/DataColumnRenderer.php

```diff
@@ -182,8 +186,18 @@ public function renderBody(ColumnInterface $column, Cell $cell, DataContext $con
             $content = '';
         }
 
+        if (is_array($column->bodyAttributes)) {
+            $attributes = $column->bodyAttributes;
+        } else {
```

Comment:
Does it worth checking if it's a callable?

Reply:
No, it's array or callable only

Reply:
But I made `is_callable()` first.